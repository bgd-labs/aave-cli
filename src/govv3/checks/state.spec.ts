import { describe, it, expect } from 'vitest';
import { MOCK_PAYLOAD } from '../mocks/payload';
import { checkStateChanges } from './state';
import { CHAIN_ID_CLIENT_MAP } from '@bgd-labs/js-utils';
import { CONFIG_CHANGE_PAYLOAD } from '../mocks/configChangePayload';

describe('state check', () => {
  it('should correctly render state diff for', async () => {
    const result = await checkStateChanges.checkProposal(
      null as any,
      MOCK_PAYLOAD.simulation,
      CHAIN_ID_CLIENT_MAP[MOCK_PAYLOAD.simulation.transaction.network_id]
    );
    expect(result).toMatchSnapshot();
  });
  it('should correctly render state diff for config change', async () => {
    const result = await checkStateChanges.checkProposal(null as any, CONFIG_CHANGE_PAYLOAD, CHAIN_ID_CLIENT_MAP[1]);
    expect(result).toMatchSnapshot();
  });
});
