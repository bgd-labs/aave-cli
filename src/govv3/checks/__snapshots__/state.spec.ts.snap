// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`state check > should correctly render state diff for 1`] = `
{
  "errors": [],
  "info": [
    "
InitializableImmutableAdminUpgradeabilityProxy (Aave USDT) at \`0x3Ed3B47Dd13EC9a98b44e6204A523E766B225811\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.ASSETS.USDT.A_TOKEN")
\`\`\`diff
@@ Slot \`0x0000000000000000000000000000000000000000000000000000000000000036\` @@
- "0x00000000000000000000000000000000000000000000000000009767a805c73e"
+ "0x000000000000000000000000000000000000000000000000000097ffa6764aa7"
@@ Slot \`0x14a553e31736f19e3e380cf55bfb2f82dfd6d880cd07235affb68d8d3e0cac4d\` @@
- "0x00000000000000000000000000000000000000000000000000000120fc65919c"
+ "0x000000000000000000000000000000000000000000000000000001b8fad61505"
\`\`\`

Dai (Dai) at \`0x6B175474E89094C44Da98b954EedeAC495271d0F\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.ASSETS.DAI.UNDERLYING, AaveV2EthereumAMM.ASSETS.DAI.UNDERLYING, AaveV3Ethereum.ASSETS.DAI.UNDERLYING")
\`\`\`diff
@@ \`balanceOf\` key \`0x464c71f6c2f760dda6093dcb91c24c39e5d6e18c\` @@
- 502,889.065017878577338563[502889065017878577338563](18 decimals)
+ 2,889.065017878577338563[2889065017878577338563](18 decimals)
@@ \`balanceOf\` key \`0x9d53cefcd5fbeb850b053a23ec6f59eb3dc76e33\` @@
- 0[0](18 decimals)
+ 500,000[500000000000000000000000](18 decimals)
@@ \`allowance\` key \`0x9d53cefcd5fbeb850b053a23ec6f59eb3dc76e33.0xc92e8bdf79f0507f65a392b0ab4667716bfe0110\` @@
- 0[0](18 decimals)
+ 115,792,089,237,316,195,423,570,985,008,687,907,853,269,984,665,640,564,039,457.584007913129639935[115792089237316195423570985008687907853269984665640564039457584007913129639935](18 decimals)
\`\`\`

InitializableImmutableAdminUpgradeabilityProxy at \`0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.POOL") with implementation LendingPool at \`0x085E34722e04567Df9E6d2c32e82fd74f3342e79\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.POOL_IMPL")
\`\`\`diff
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).liquidityIndex\` @@
- 1.125972900043692179606694722[1125972900043692179606694722](27 decimals)
+ 1.125991030272021514250652312[1125991030272021514250652312](27 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).variableBorrowIndex\` @@
- 1.185069240973920592952379763[1185069240973920592952379763](27 decimals)
+ 1.185097276944910539318953337[1185097276944910539318953337](27 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).currentLiquidityRate\` @@
- 10.0751473401236648012460614%[100751473401236648012460614](25 decimals)
+ 12.3029815053899026525991182%[123029815053899026525991182](25 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).currentVariableBorrowRate\` @@
- 14.8027638325434642033107448%[148027638325434642033107448](25 decimals)
+ 18.0113117017905438863598858%[180113117017905438863598858](25 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).currentStableBorrowRate\` @@
- 19.8027638325434642033107448%[198027638325434642033107448](25 decimals)
+ 23.0113117017905438863598858%[230113117017905438863598858](25 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).lastUpdateTimestamp\` @@
- 1705396271
+ 1705401311
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).liquidityIndex\` @@
- 1.149223734302009162656035865[1149223734302009162656035865](27 decimals)
+ 1.149227808508467355639962102[1149227808508467355639962102](27 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).variableBorrowIndex\` @@
- 1.228776684865256100554716625[1228776684865256100554716625](27 decimals)
+ 1.228783878292595972201993472[1228783878292595972201993472](27 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).currentLiquidityRate\` @@
- 4.5670276364018875818860783%[45670276364018875818860783](25 decimals)
+ 3.6453252711281215796169147%[36453252711281215796169147](25 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).currentVariableBorrowRate\` @@
- 7.541484132616072667154102%[75414841326160726671541020](25 decimals)
+ 5.9996093895016005528209445%[59996093895016005528209445](25 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).currentStableBorrowRate\` @@
- 13.541484132616072667154102%[135414841326160726671541020](25 decimals)
+ 11.9998697965005335176069815%[119998697965005335176069815](25 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).lastUpdateTimestamp\` @@
- 1705398863
+ 1705401311
\`\`\`

InitializableImmutableAdminUpgradeabilityProxy at \`0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV3Ethereum.POOL") with implementation Pool at \`0x5FAab9E1adbddaD0a08734BE8a52185Fd6558E14\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV3Ethereum.POOL_IMPL")
\`\`\`diff
@@ Slot \`0xed960c71bd5fa1333658850f076b35ec5565086b606556c3dd36a916b43ddf21\` @@
- "0x000000000027c683ca3377fb1f1693f7000000000359f41baa6a46f058e50aee"
+ "0x00000000002771189d860f9c4c5d9a01000000000359f41db929cf9775960caa"
@@ Slot \`0xed960c71bd5fa1333658850f076b35ec5565086b606556c3dd36a916b43ddf22\` @@
- "0x0000000000315e12389ab3a83d7bef4e0000000003631ce935afe1cefdf4473d"
+ "0x00000000003128f35faf2f87fbfe5bc00000000003631cebca71407f804fda4c"
@@ Slot \`0xed960c71bd5fa1333658850f076b35ec5565086b606556c3dd36a916b43ddf23\` @@
- "0x00000000000000000000030065a65bc700000000003e043fc9ee429d6074fe9c"
+ "0x00000000000000000000030065a65bdf00000000003dffd28d2ff79ab0553250"
@@ Slot \`0xed960c71bd5fa1333658850f076b35ec5565086b606556c3dd36a916b43ddf28\` @@
- "0x00000000000000000000000000000000000000000000000000000009943615e8"
+ "0x000000000000000000000000000000000000000000000000000000099459516a"
\`\`\`

InitializableImmutableAdminUpgradeabilityProxy (Aave v3 USDC) at \`0x98C23E9d8f34FEFb1B7BD6a91B7FF122F4e16F5c\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV3Ethereum.ASSETS.USDC.A_TOKEN") with implementation AToken at \`0x7EfFD7b47Bfd17e52fB7559d3f924201b9DbfF3d\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV3Ethereum.DEFAULT_A_TOKEN_IMPL_REV_1")
\`\`\`diff
@@ Slot \`0x0000000000000000000000000000000000000000000000000000000000000036\` @@
- "0x0000000000000000000000000000000000000000000000000002046ddd0d5441"
+ "0x0000000000000000000000000000000000000000000000000002069bf1c3af1c"
@@ Slot \`0x14a553e31736f19e3e380cf55bfb2f82dfd6d880cd07235affb68d8d3e0cac4d\` @@
- "0x0000000003596234d01403e611c0216d0000000000000000000000b4c6d30f4f"
+ "0x000000000359f41db929cf9775960caa0000000000000000000002e2db896a2a"
\`\`\`

unknown contract name at \`0x9CE312c09AB6cA0cc4cb7d7CECA975729F1ffB6a\`
\`\`\`diff
@@ Slot \`0x0000000000000000000000000000000000000000000000000000000000000000\` @@
- "0x0000000000000000000000000000000000000000000000000000000000000000"
+ "0xd995ff6cf3e2498dd70cd06dd7b3f4c06840ba62f7b1ff3c20b766f1f572a8d2"
\`\`\`

unknown contract name at \`0x9d53CeFcd5fbeb850B053a23Ec6F59eb3Dc76E33\`
\`\`\`diff
@@ Slot \`0x0000000000000000000000000000000000000000000000000000000000000000\` @@
- "0x0000000000000000000000000000000000000000000000000000000000000000"
+ "0x5277a0a374a2530b7c07818355e3f7fb438196e1c8b1353925e712542064af9e"
\`\`\`

FiatTokenProxy (USDC) at \`0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.ASSETS.USDC.UNDERLYING, AaveV2EthereumAMM.ASSETS.USDC.UNDERLYING, AaveV3Ethereum.ASSETS.USDC.UNDERLYING") with implementation FiatTokenV2_2 at \`0x43506849D7C04F9138D1A2050bbF3A0c054402dd\`
\`\`\`diff
@@ Slot \`0x368c3f5f03e5634b3e4381c9c3caac98f4d254c7027fd14b53436c90d060fef4\` @@
- "0x000000000000000000000000000000000000000000000000000038207ce5ee96"
+ "0x00000000000000000000000000000000000000000000000000003a6354a5488f"
@@ Slot \`0x436e89b6ed03251065e7982780d1be43053fbe0f887317733835576215f6bcca\` @@
- "0x000000000000000000000000000000000000000000000000000017daed464f11"
+ "0x0000000000000000000000000000000000000000000000000000163b10c74d18"
@@ Slot \`0x4f5f42308eb505bb367768d921d9ad787d6b433ee8d3b15ce32cfeb0bed816d0\` @@
- "0x0000000000000000000000000000000000000000000000000000018bcfe56f2b"
+ "0x000000000000000000000000000000000000000000000000000000000000072b"
@@ Slot \`0xd3f86232aca8cfc941e263eb25877a23f44585d3bb6ccab86bcc91db51469ac5\` @@
- "0x0000000000000000000000000000000000000000000000000000000000000000"
+ "0x000000000000000000000000000000000000000000000000000000e8d4a51000"
@@ Slot \`0xd9e23465a7cab947bc7f4bd993ff45b2b649507a2de4042ad75758d79d7e7256\` @@
- "0x0000000000000000000000000000000000000000000000000000000000000000"
+ "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
\`\`\`

InitializableImmutableAdminUpgradeabilityProxy (Aave v2 USDC) at \`0xBcca60bB61934080951369a648Fb03DF4F96263C\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.ASSETS.USDC.A_TOKEN")
\`\`\`diff
@@ Slot \`0x0000000000000000000000000000000000000000000000000000000000000036\` @@
- "0x0000000000000000000000000000000000000000000000000000f84a2f718226"
+ "0x0000000000000000000000000000000000000000000000000000f6d93284198f"
@@ Slot \`0x14a553e31736f19e3e380cf55bfb2f82dfd6d880cd07235affb68d8d3e0cac4d\` @@
- "0x000000000000000000000000000000000000000000000000000001715450e483"
+ "0x0000000000000000000000000000000000000000000000000000000057637bec"
\`\`\`

InitializableImmutableAdminUpgradeabilityProxy at \`0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.DEFAULT_INCENTIVES_CONTROLLER")
\`\`\`diff
@@ \`assets\` key \`0x3ed3b47dd13ec9a98b44e6204a523e766b225811 (symbol: aUSDT).lastUpdateTimestamp\` @@
- 1705398863
+ 1705401311
@@ \`assets\` key \`0xbcca60bb61934080951369a648fb03df4f96263c (symbol: aUSDC).lastUpdateTimestamp\` @@
- 1705396271
+ 1705401311
@@ \`assets\` key \`0xbcca60bb61934080951369a648fb03df4f96263c (symbol: aUSDC).users.0x5300a1a15135ea4dc7ad5a167152c01efc9b192a\` @@
- 0
+ 35777739039773953913685859
\`\`\`

TransparentUpgradeableProxy at \`0xdAbad81aF85554E9ae636395611C58F7eC1aAEc5\`[:ghost:](https://github.com/bgd-labs/aave-address-book "GovernanceV3Ethereum.PAYLOADS_CONTROLLER") with implementation PayloadsController at \`0x7222182cB9c5320587b5148BF03eeE107AD64578\`
\`\`\`diff
@@ Slot \`0xa7d0f7195d52522be008ca0e9c182cb8d5cdec7c4327b16f8f80417732546566\` @@
- "0x0065a5097a0065a02ba302013765a685a401622c060e5d700d9ad89413363a91"
+ "0x0065a5097a0065a02ba303013765a685a401622c060e5d700d9ad89413363a91"
@@ Slot \`0xa7d0f7195d52522be008ca0e9c182cb8d5cdec7c4327b16f8f80417732546567\` @@
- "0x000000000000000000093a8000000151800065ce502300000000000000000000"
+ "0x000000000000000000093a8000000151800065ce502300000000000065a65bdf"
\`\`\`

TetherToken (Tether) at \`0xdAC17F958D2ee523a2206206994597C13D831ec7\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.ASSETS.USDT.UNDERLYING, AaveV2EthereumAMM.ASSETS.USDT.UNDERLYING, AaveV3Ethereum.ASSETS.USDT.UNDERLYING")
\`\`\`diff
@@ \`balances\` key \`0x3ed3b47dd13ec9a98b44e6204a523e766b225811\` @@
- 37,673,056.276055[37673056276055](6 decimals)
+ 38,423,056.276055[38423056276055](6 decimals)
@@ \`balances\` key \`0x464c71f6c2f760dda6093dcb91c24c39e5d6e18c\` @@
- 750,000[750000000000](6 decimals)
+ 0[0](6 decimals)
\`\`\`
",
  ],
  "warnings": [],
}
`;

exports[`state check > should correctly render state diff for config change 1`] = `
{
  "errors": [],
  "info": [
    "
InitializableImmutableAdminUpgradeabilityProxy at \`0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.POOL") with implementation LendingPool at \`0x085E34722e04567Df9E6d2c32e82fd74f3342e79\`[:ghost:](https://github.com/bgd-labs/aave-address-book "AaveV2Ethereum.POOL_IMPL")
\`\`\`diff
@@ \`_reserves\` key \`0x0000000000085d4780b73119b644ae5ecd22b376 (symbol: TUSD).configuration.data\` @@
- 184283194582935181459456
+ 184449215279598567424000
@@ \`_reserves\` key \`0x0000000000085d4780b73119b644ae5ecd22b376 (symbol: TUSD).configuration.data_decoded.reserveFactor\` @@
- 99.9%[9990](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x03ab458634910aad20ef5f1c8ee96f1d6ac54919 (symbol: RAI).configuration.data\` @@
- 182623275799432407285760
+ 184449503462729652895744
@@ \`_reserves\` key \`0x03ab458634910aad20ef5f1c8ee96f1d6ac54919 (symbol: RAI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x056fd409e1d7a124bd7017459dfea2f387b6d5cd (symbol: GUSD).configuration.data\` @@
- 36893848998339246292992
+ 46117221035194022100992
@@ \`_reserves\` key \`0x056fd409e1d7a124bd7017459dfea2f387b6d5cd (symbol: GUSD).configuration.data_decoded.reserveFactor\` @@
- 20%[2000](2 decimals)
+ 25%[2500](2 decimals)
@@ \`_reserves\` key \`0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e (symbol: YFI).configuration.data\` @@
- 182623275846677047869440
+ 184449503509974293479424
@@ \`_reserves\` key \`0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e (symbol: YFI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x0d8775f648430679a709e98d2b0cb6250d2887ef (symbol: BAT).configuration.data\` @@
- 182622987616300896157696
+ 184449215279598141767680
@@ \`_reserves\` key \`0x0d8775f648430679a709e98d2b0cb6250d2887ef (symbol: BAT).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x0f5d2fb29fb7d3cfee444a200298f468908cc942 (symbol: MANA).configuration.data\` @@
- 182623275846677047869440
+ 184449503509974293479424
@@ \`_reserves\` key \`0x0f5d2fb29fb7d3cfee444a200298f468908cc942 (symbol: MANA).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x111111111117dc0aa78b770fa6a738034120c302 (symbol: 1INCH).configuration.data\` @@
- 182622987615656651063296
+ 184449215278953896673280
@@ \`_reserves\` key \`0x111111111117dc0aa78b770fa6a738034120c302 (symbol: 1INCH).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x1494ca1f11d487c2bbe4543e90080aeba4ba3c2b (symbol: DPI).configuration.data\` @@
- 182622987616300896157696
+ 184449215279598141767680
@@ \`_reserves\` key \`0x1494ca1f11d487c2bbe4543e90080aeba4ba3c2b (symbol: DPI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x1f9840a85d5af5bf1d1762f925bdaddc4201f984 (symbol: UNI).configuration.data\` @@
- 182622987615871490850816
+ 184449215279168736460800
@@ \`_reserves\` key \`0x1f9840a85d5af5bf1d1762f925bdaddc4201f984 (symbol: UNI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x2260fac5e5542a773aa44fbcfedf7c193bc2c599 (symbol: WBTC).configuration.data\` @@
- 55340594805996352183328
+ 64563966842851127991328
@@ \`_reserves\` key \`0x2260fac5e5542a773aa44fbcfedf7c193bc2c599 (symbol: WBTC).configuration.data_decoded.reserveFactor\` @@
- 30%[3000](2 decimals)
+ 35%[3500](2 decimals)
@@ \`_reserves\` key \`0x408e41876cccdc0f92210600ef50372656052a38 (symbol: REN).configuration.data\` @@
- 182622987616300896157696
+ 184449215279598141767680
@@ \`_reserves\` key \`0x408e41876cccdc0f92210600ef50372656052a38 (symbol: REN).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b (symbol: CVX).configuration.data\` @@
- 182622987615656651063296
+ 184449215278953896673280
@@ \`_reserves\` key \`0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b (symbol: CVX).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x4fabb145d64652a948d72533023f6e7a623c7c53 (symbol: BUSD).configuration.data\` @@
- 184283482766066266931200
+ 184449503462729652895744
@@ \`_reserves\` key \`0x4fabb145d64652a948d72533023f6e7a623c7c53 (symbol: BUSD).configuration.data_decoded.reserveFactor\` @@
- 99.9%[9990](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x514910771af9ca656af840dff83e8264ecf986ca (symbol: LINK).configuration.data\` @@
- 55340453506416971350016
+ 64563825543271747158016
@@ \`_reserves\` key \`0x514910771af9ca656af840dff83e8264ecf986ca (symbol: LINK).configuration.data_decoded.reserveFactor\` @@
- 30%[3000](2 decimals)
+ 35%[3500](2 decimals)
@@ \`_reserves\` key \`0x57ab1ec28d129707052df4df418d58a2d46d5f51 (symbol: sUSD).configuration.data\` @@
- 55340597575648425279488
+ 64563969612503201087488
@@ \`_reserves\` key \`0x57ab1ec28d129707052df4df418d58a2d46d5f51 (symbol: sUSD).configuration.data_decoded.reserveFactor\` @@
- 30%[3000](2 decimals)
+ 35%[3500](2 decimals)
@@ \`_reserves\` key \`0x5f98805a4e8be255a32880fdec7f6728c6568ba0 (symbol: LUSD).configuration.data\` @@
- 46117225538793649471488
+ 55340597575648425279488
@@ \`_reserves\` key \`0x5f98805a4e8be255a32880fdec7f6728c6568ba0 (symbol: LUSD).configuration.data_decoded.reserveFactor\` @@
- 25%[2500](2 decimals)
+ 30%[3000](2 decimals)
@@ \`_reserves\` key \`0x6b175474e89094c44da98b954eedeac495271d0f (symbol: DAI).configuration.data\` @@
- 46117225583461879520588
+ 55340597620316655328588
@@ \`_reserves\` key \`0x6b175474e89094c44da98b954eedeac495271d0f (symbol: DAI).configuration.data_decoded.reserveFactor\` @@
- 25%[2500](2 decimals)
+ 30%[3000](2 decimals)
@@ \`_reserves\` key \`0x853d955acef822db058eb8505911ed77f175b99e (symbol: FRAX).configuration.data\` @@
- 55340597575648425279488
+ 64563969612503201087488
@@ \`_reserves\` key \`0x853d955acef822db058eb8505911ed77f175b99e (symbol: FRAX).configuration.data_decoded.reserveFactor\` @@
- 30%[3000](2 decimals)
+ 35%[3500](2 decimals)
@@ \`_reserves\` key \`0x8798249c2e607446efb7ad49ec89dd1865ff4272 (symbol: xSUSHI).configuration.data\` @@
- 182622987616300896157696
+ 184449215279598141767680
@@ \`_reserves\` key \`0x8798249c2e607446efb7ad49ec89dd1865ff4272 (symbol: xSUSHI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x8e870d67f660d95d5be530380d0ec0bd388289e1 (symbol: USDP).configuration.data\` @@
- 36893853501938873663488
+ 46117225538793649471488
@@ \`_reserves\` key \`0x8e870d67f660d95d5be530380d0ec0bd388289e1 (symbol: USDP).configuration.data_decoded.reserveFactor\` @@
- 20%[2000](2 decimals)
+ 25%[2500](2 decimals)
@@ \`_reserves\` key \`0x956f47f50a910163d8bf957cf5846d573e7f87ca (symbol: FEI).configuration.data\` @@
- 182623275846677047869440
+ 184449503509974293479424
@@ \`_reserves\` key \`0x956f47f50a910163d8bf957cf5846d573e7f87ca (symbol: FEI).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2 (symbol: unknown).configuration.data\` @@
- 182622987615227245756416
+ 184449215278524491366400
@@ \`_reserves\` key \`0x9f8f72aa9304c8b593d555f12ef6589cc3a579a2 (symbol: unknown).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).configuration.data\` @@
- 46117222205976910634816
+ 55340594242831686442816
@@ \`_reserves\` key \`0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48 (symbol: USDC).configuration.data_decoded.reserveFactor\` @@
- 25%[2500](2 decimals)
+ 30%[3000](2 decimals)
@@ \`_reserves\` key \`0xa693b19d2931d498c5b318df961919bb4aee87a5 (symbol: UST).configuration.data\` @@
- 182623272421732686757888
+ 184449500085029932367872
@@ \`_reserves\` key \`0xa693b19d2931d498c5b318df961919bb4aee87a5 (symbol: UST).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xba100000625a3754423978a60c9317c58a424e3d (symbol: BAL).configuration.data\` @@
- 182622987615441902698496
+ 184449215278739148308480
@@ \`_reserves\` key \`0xba100000625a3754423978a60c9317c58a424e3d (symbol: BAL).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f (symbol: SNX).configuration.data\` @@
- 182622987615227154333696
+ 184449215278524399943680
@@ \`_reserves\` key \`0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f (symbol: SNX).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 (symbol: WETH).configuration.data\` @@
- 46117225583891369697338
+ 55340597620746145505338
@@ \`_reserves\` key \`0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2 (symbol: WETH).configuration.data_decoded.reserveFactor\` @@
- 25%[2500](2 decimals)
+ 30%[3000](2 decimals)
@@ \`_reserves\` key \`0xc18360217d8f7ab5e7c516566761ea12ce7f9d72 (symbol: ENS).configuration.data\` @@
- 182622987615441902698496
+ 184449215278739148308480
@@ \`_reserves\` key \`0xc18360217d8f7ab5e7c516566761ea12ce7f9d72 (symbol: ENS).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xd46ba6d942050d489dbd938a2c909a5d5039a161 (symbol: AMPL).configuration.data\` @@
- 184283480232791476535296
+ 184449500929454862499840
@@ \`_reserves\` key \`0xd46ba6d942050d489dbd938a2c909a5d5039a161 (symbol: AMPL).configuration.data_decoded.reserveFactor\` @@
- 99.9%[9990](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xd533a949740bb3306d119cc777fa900ba034cd52 (symbol: CRV).configuration.data\` @@
- 182622987615441994121216
+ 184449215278739239731200
@@ \`_reserves\` key \`0xd533a949740bb3306d119cc777fa900ba034cd52 (symbol: CRV).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).configuration.data\` @@
- 46117222161093928943616
+ 55340594197948704751616
@@ \`_reserves\` key \`0xdac17f958d2ee523a2206206994597c13d831ec7 (symbol: USDT).configuration.data_decoded.reserveFactor\` @@
- 25%[2500](2 decimals)
+ 30%[3000](2 decimals)
@@ \`_reserves\` key \`0xdd974d5c2e2928dea5f71b9825b8b646686bd200 (symbol: KNC).configuration.data\` @@
- 182623275846677047869440
+ 184449503509974293479424
@@ \`_reserves\` key \`0xdd974d5c2e2928dea5f71b9825b8b646686bd200 (symbol: KNC).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xe41d2489571d322189246dafa5ebde1f4699f498 (symbol: ZRX).configuration.data\` @@
- 182622987616300948258816
+ 184449215279598193868800
@@ \`_reserves\` key \`0xe41d2489571d322189246dafa5ebde1f4699f498 (symbol: ZRX).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
@@ \`_reserves\` key \`0xf629cbd94d3791c9250152bd8dfbdf380e2a3b9c (symbol: ENJ).configuration.data\` @@
- 182623275846677047869440
+ 184449503509974293479424
@@ \`_reserves\` key \`0xf629cbd94d3791c9250152bd8dfbdf380e2a3b9c (symbol: ENJ).configuration.data_decoded.reserveFactor\` @@
- 99%[9900](2 decimals)
+ 99.99%[9999](2 decimals)
\`\`\`

TransparentUpgradeableProxy at \`0xdAbad81aF85554E9ae636395611C58F7eC1aAEc5\`[:ghost:](https://github.com/bgd-labs/aave-address-book "GovernanceV3Ethereum.PAYLOADS_CONTROLLER") with implementation PayloadsController at \`0x7222182cB9c5320587b5148BF03eeE107AD64578\`
\`\`\`diff
@@ Slot \`0x35cf9ccc5fb50786824d0efe505d33216d9658f34614e7c25f0d5baeb2b0c672\` @@
- "0x0065f066320065ef4eaf020157ab7ee15ce5ecacb1ab84ee42d5a9d0d8112922"
+ "0x0065f066320065ef4eaf030157ab7ee15ce5ecacb1ab84ee42d5a9d0d8112922"
@@ Slot \`0x35cf9ccc5fb50786824d0efe505d33216d9658f34614e7c25f0d5baeb2b0c673\` @@
- "0x000000000000000000093a80000001518000661d732f00000000000000000000"
+ "0x000000000000000000093a80000001518000661d732f00000000000065f1b88b"
\`\`\`
",
  ],
  "warnings": [],
}
`;
